fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/RebuildIT.java

```diff
@@ -57,28 +54,20 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RebuildIT.class);
 
-    private static String origStorageRootDir;
-    private static String origWorkDir;
+    private static final String OCFL_ROOT_PROP = "fcrepo.ocfl.root";
 
     private OcflRepository ocflRepository;
 
     private IndexBuilder indexBuilder;
 
     @BeforeClass
     public static void beforeClass() {
-        // Save the pre-test System Property values
-        origStorageRootDir = getProperty(OCFL_STORAGE_ROOT_DIR_KEY);
-        origWorkDir = getProperty(OCFL_WORK_DIR_KEY);
-
-        System.setProperty(OCFL_STORAGE_ROOT_DIR_KEY, "target/test-classes/test-rebuild-ocfl/ocfl-root");
-        System.setProperty(OCFL_WORK_DIR_KEY, "target/test-classes/test-rebuild-ocfl/ocfl-work");
+        System.setProperty(OCFL_ROOT_PROP, "target/test-classes/test-rebuild-ocfl/ocfl-root");
     }
 
     @AfterClass
     public static void afterClass() {
-        // Restore pre-test System Property values
-        System.setProperty(OCFL_STORAGE_ROOT_DIR_KEY, origStorageRootDir);
-        System.setProperty(OCFL_WORK_DIR_KEY, origWorkDir);
+        System.clearProperty(OCFL_ROOT_PROP);
```

Comment:
Is there a potential need to reset the value of this property to what it was before this test?

Reply:
Since the default is to NOT have that property set, any test that is setting the property should clear it when they're done.